demisto_sdk/__main__.py

```diff
@@ -423,30 +430,63 @@ def prepare_content(ctx, **kwargs):
     """
     This command is used to prepare the content to be used in the platform.
     """
-    if click.get_current_context().info_name == "unify":
-        kwargs["unify_only"] = True
+    assert sum([bool(kwargs["all"]), bool(kwargs["input"])]) == 1, (
+        "Exactly one of the '-a' or '-i' parameters must " "be provided. "
+    )
 
-    check_configuration_file("unify", kwargs)
-    # Input is of type Path.
-    kwargs["input"] = str(kwargs["input"])
-    file_type = find_type(kwargs["input"])
-    if marketplace := kwargs.get("marketplace"):
-        os.environ[ENV_DEMISTO_SDK_MARKETPLACE] = marketplace.lower()
-    if file_type == FileType.GENERIC_MODULE:
-        from demisto_sdk.commands.prepare_content.generic_module_unifier import (
-            GenericModuleUnifier,
+    if kwargs["all"]:
+        content_DTO = from_path()
+        output_path = kwargs.get("output", ".") or "."
+        content_DTO.dump(
+            dir=Path(output_path, "prepare-content-tmp"),
+            marketplace=parse_marketplace_kwargs(kwargs),
         )
+        return 0
 
-        # pass arguments to GenericModule unifier and call the command
-        generic_module_unifier = GenericModuleUnifier(**kwargs)
-        generic_module_unifier.merge_generic_module_with_its_dashboards()
-    else:
-        PrepareUploadManager.prepare_for_upload(**kwargs)
+    inputs = []
+    if input_ := kwargs["input"]:
+        inputs = input_.split(",")
+
+    if output_path := kwargs["output"]:
+        if "." in Path(output_path).name:  # check if the output path is a file
```

Comment:
```suggestion
        if Path(output_path).is_file():
```

Reply:
At this point, the file doesn't exist yet, so using `Path.is_fiile()` will always return false.